notebooks/kubeflow_pipelines/pipelines/solutions/kfp_pipeline_vertex_automl.ipynb

```diff
@@ -0,0 +1,373 @@
+{
```

Comment:
<p>typo: remove "a" in objective #3</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/153/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Done!</p>